### PR TITLE
Updating API Repository

### PR DESCRIPTION
**Pull request by leeservice**

### CODE REVIEW

**Review by david-andrew-k — APPROVED** [review]
Looks good so far, could we look at what the banner files are and if they are necessary?

**Comment by leeservice** [comment]
I'll delete these files as they're unneeded for now.

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1 @@
+mvn clean test
```
Comment (ccurran01) [inline .husky/pre-commit:1]: This seems like it will eventually start to run the tests. Is that something that we want?

**Review by ccurran01 — COMMENTED** [review]
The .idea and target files are in the `.gitignore` file but somehow seem to have slipped through. Best get those cleared up. 

Not sure what the banner files are either but I imagine they're safe to get rid of.

**Comment by leeservice** [comment]
This should be all fixes for the /target and .idea folders.

**Review by leeservice — COMMENTED** [review]
Fixed

**Review by ccurran01 — APPROVED** [review]

**Pull request closed without merging**